states/games/snake.mod.py

```diff
@@ -158,27 +150,19 @@ def export(self):
 
         return image
 
-    def receive(self):
-        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
-            s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-            s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)
-            s.bind(("0.0.0.0", 1029))
-            s.listen()
-            while not self.state.killed:
-                conn, addr = s.accept()
-                player = Player(conn, addr, self.state)
-                self.players.append(player)
-                self.apples.append(Apple())
-
 
 class State(BaseState):
     def __init__(self):
         super().__init__()
         self.name = "snake"
         self.index = 8
         self.delay = 5
+        self.is_game = True
```

Comment:
Vraag: hoe wordt nu bepaald welke game wordt gestart?

Reply:
Via de normale check functie, deze property geeft alleen maar aan dat players deze game kunnen joinen via socket.io

Reply:
Maar die check bevat vaak iets als 'zijn er players connected'. Dus dan zouden alle games opeens True gaan geven en krijg je dus een random game die gestart wordt. Omdat we nu maar 1 poort hebben zal de client iets van een game naam door moeten sturen die gestart moet worden.

Reply:
Dat doet de client al, zie states/socket.py. De client kan een join, game id event geven om games te joinen